composer.json

```diff
@@ -18,7 +18,7 @@
     "require": {
         "php": "^5.5 || ^7.0",
         "doctrine/phpcr-bundle": "^1.1",
-        "doctrine/phpcr-odm": "^1.1",
+        "doctrine/phpcr-odm": "^1.1 || ^2.0",
```

Comment:
If I understand Composer correctly, the `@dev` flag applies to the complete constraint: `(^1.1 || ^2.0)@dev`.

Also, the dev flag does apply to root-only. So packages needing PHPCR-ODM 2.0 would need to set the minimum stability to dev anyway.

If we want to test against this, I suggest adding a dev-dependency test job like this: https://github.com/symfony-cmf/routing-bundle/blob/master/.travis.yml#L35